pkg/servicedisc/client.go

```diff
@@ -112,9 +106,9 @@ func (c *HTTPClient) Auth(ctx context.Context) (*httpauth.Client, error) {
 	return auth, nil
 }
 
-// Services calls 'GET /api/services'.
+// Services calls 'GET /api/entries'.
 func (c *HTTPClient) Services(ctx context.Context, quantity int) (out []Service, err error) {
-	url, err := c.addr("/api/services", c.entry.Type, quantity)
+	url, err := c.addr("/api/entries", c.entry.Type, quantity)
```

Comment:
Please do not change the endpoint for the regsitration/deregistration.

Reply:
Done!